### PR TITLE
Widget image captioning

### PR DESCRIPTION
**Pull request by mishig25**
Widget side of Image Captioning pipeline

input/request: identical to ImageClassificationWidget
`const requestBody = { file }; // uploaded image`

output/response: identical to TextGenerationWidget
`[ { "generated_text": "..." } ]`

### CODE REVIEW

File: interfaces/Types.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ export enum PipelineType {
 	"audio-source-separation" = "audio-source-separation",
 	"voice-activity-detection" = "voice-activity-detection",
 	/// computer vision
+	"image-captioning" = "image-captioning",
```
Comment (julien-c) [inline interfaces/Types.ts:26]: `image-to-text` if we expect other more general models that do this? What do you think?

Comment (mishig25) [reply interfaces/Types.ts:26]: Totally. Will update the definition to `image-to-text`

**Comment by mishig25** [comment]
Is there anything I can do for the `Netlify deploy` error [here](https://github.com/huggingface/huggingface_hub/pull/115/checks?check_run_id=2856502996)?

**Comment by julien-c** [comment]
weird, would need to investigate what's happening 

We're using https://docs.netlify.com/cli/get-started/#manual-deploys

**Comment by mishig25** [comment]
Screenshots:
Case 1: Initial state             |  Case 2: Success 
:-------------------------:|:-------------------------:
Screenshot from 2021-06-20 08-13-40 | Screenshot from 2021-06-20 08-19-19

Case 3: Fail (wrong file type)  
:-------------------------:
Screenshot from 2021-06-20 08-21-28

**Comment by osanseviero** [comment]
Why was this PR closed?

**Pull request closed without merging**